collateral.md

```diff
@@ -47,21 +56,192 @@ module COLLATERAL
 
     syntax FlipStep ::= "kick" Address Address Int Int Int
  // ------------------------------------------------------
+    rule <k> Flip ILK . kick USR GAL TAB LOT BID
+          => Vat . flux ILK MSGSENDER THIS LOT
+          ~> KICKS +Int 1 ... </k>
+         <msg-sender> MSGSENDER </msg-sender>
+         <this> THIS </this>
+         <currentTime> NOW </currentTime>
+         <flip-ilk> ILK </flip-ilk>
+         <flip-tau> TAU </flip-tau>
+         <flip-kicks> KICKS => KICKS +Int 1 </flip-kicks>
+         <flip-bids>... .Map =>
+           KICKS +Int 1 |-> Bid(...
+                             bid: BID,
+                             lot: LOT,
+                             guy: MSGSENDER,
+                             tic: 0,
+                             end: NOW +Int TAU,
+                             usr: USR,
+                             gal: GAL,
+                             tab: TAB)
+         ...</flip-bids>
 
     syntax FlipStep ::= "tick" Int
  // ------------------------------
+    rule <k> Flip ILK . tick ID => Flip ILK . exception ... </k>
```

Comment:
I agree, it would be nice to ensure the semantics is total via `owise` rules, and only define the positive cases if possible.

If it becomes really difficult to define the positive cases, we can re-evaluate this decision.